### ISSUE TITLE
Duplication with the last version 1.10.2-1.4.1

### ISSUE THREAD

**Issue by Gohu29** [rwtema/extrautilities_old_issues#1460]
Builder's wand don't use items in inventory. I can build indefinitely with it.

SpongeAPI: 5.2.0-SNAPSHOT-a3257a0
SpongeForge: 1.10.2-2254-5.2.0-BETA-2236
Minecraft Forge: 12.18.3.2254

**Comment by ArukaAlter** [comment]
Can be reproduced with only ExtraUtilities 1.4.1 and Forge
https://gfycat.com/UnpleasantWelcomeDolphin

**Comment by jawsawn** [comment]
I can confirm it exists.

**Comment by fael13** [comment]
Me too. 

Minecraft 1.10.2
forge 12.18.3.2281
Modpack SKyfactorio 3 modpack + quark.

**Comment by fucnknown** [comment]
+1

**Comment by jawsawn** [comment]
Seeing that this important dupe bug hasnt fixed uet is somewhat sad...

Only thing we can do is disable its recipe

**Comment by fucnknown** [comment]
Agreed, that's what we had to do. (And then went around confiscating any wand that was on player's inventories)

**Comment by ktole1999** [comment]
or you could not run the latest version until the issue is resolved

**Comment by fucnknown** [comment]
Well, regrettably that's the version included with SkyFactory3, and it's a hassle to force every player to downgrade the mod in the pack :/

**Comment by ktole1999** [comment]
The latest version of pack, correct?



Sent via the Samsung Galaxy S® 6, an AT&T 4G LTE smartphone


-------- Original message --------

**Comment by fucnknown** [comment]
Latest stable release, [3.0.9](https://www.feed-the-beast.com/projects/ftb-presents-skyfactory-3/files), Next to last release.

**Comment by Flexico** [comment]
What's the latest version that doesn't have the bug? I've tried 1.4.0-1.4.4 and they all have it.

**Comment by Flexico** [comment]
Well that was a question quickly answered. XD For others looking, it's 1.3.3.
